src/Momento.Sdk/Internal/DataGrpcManager.cs

```diff
@@ -168,7 +168,7 @@ public async Task<_ListLengthResponse> ListLengthAsync(_ListLengthRequest reques
     }
 }
 
-internal class DataGrpcManager : IDisposable
+public class DataGrpcManager : IDisposable
```

Comment:
Motivation for visibility modifier change?

Reply:
Incubating repo needs access to this